services/wfm/src/main/java/org/openkilda/wfm/topology/nbworker/bolts/RouterBolt.java

```diff
@@ -83,4 +85,9 @@ public void declareOutputFields(OutputFieldsDeclarer declarer) {
         declarer.declareStream(StreamType.ISL.toString(), new Fields("request", "correlationId"));
         declarer.declareStream(StreamType.FLOW.toString(), new Fields("request", "correlationId"));
     }
+
+    @Override
+    protected Logger getLog() {
```

Comment:
there are already defined loggers in nbworker topology, could you replace old ones?

Reply:
my fail, fixing it.